scripts/install_on_docker.sh

```diff
@@ -0,0 +1,199 @@
+# vim: set ft=dockerfile:
```

Comment:
This script seems redundant with the Dockerfile for CentOS 6, and doesn't seem to be used anywhere. Can we remove it?

Reply:
Done!